### ISSUE TITLE
eleven_multilingual_v1

### ISSUE THREAD

**Issue by W4RI0CK** [hparcells/rtvc#5]
Is that possible to play the voice with the eleven_multilingual_v1?

**Comment by hparcells** [comment]
Yes, there does seem to be a new option to select a voice model on the `/v1/text-to-speech/{voice_id}` endpoint, as well as endpoints to view all models ElevenLabs has to offer. A new option will be available to select soon.
